### PR TITLE
Awesomebooks plain js

### PR DESCRIPTION
**Pull request by abeltsew**
Dear Reviewer,

In this PR we have implemented the required below features in the Awsom books Js plain exercise.

- [x] Create a collection that keeps a list of books in local Storage.

- [x] Create a function to add a new book to the collection, with title and author.

- [x] Create a function to remove a book from the collection and reflect changes.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      const id = remove.getAttribute('book_id');
+      afterRefresh(() => removeBook(id));
+    });
+
+    container.appendChild(book);
+  });
+}
+
+// Inital render
+render();
```
Comment (mckent05) [inline index.js]: - To make your code more readable and easier to understand, I suggest you attach this function to an event listener. In this case, DOMContentLoaded would be appropriate. Please fix this 🙏

File: index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable array-callback-return */
```
Comment (mckent05) [inline index.js]: - Please remove all disabled linters from your codebase, remember linters help to ensure that we write code that is clean and easy to maintain. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @abeltsew, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job rendering the books dynamically 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (mahabubx7) [reply index.js]: image
We followed this :eyes:

File: index.js (hunk 1 above)
Comment (mahabubx7) [reply index.js]: I'm on it ... :rocket:

**Review by DammyShittu — APPROVED** [review]
Hi @abeltsew and @mahabubx7,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mahabubx7** [comment]
:congratulations: Congratulations! @abeltsew.
We made it :grinning: 

Now, merge it :rocket:

**Pull request merged**